includes/settings.php

```diff
@@ -488,6 +508,56 @@ public function do_settings_sections( $page, $columns = 2 ) {
 	}
 
 	/**
+	 * Generate the help tab content
+	 *
+	 * @since NEXT
+	 *
+	 * @return mixed
+	 */
+	public function generate_help_tab_content() {
+
+		$language  = get_locale();
+		$parts     = explode( '_', $language );
+		$subdomain = ! empty( $parts[1] ) ? strtolower( $parts[1] ) : strtolower( $language );
+
+		// Overrides
+		switch ( $subdomain ) {
+
+			case '' :
+
+				$subdomain = 'www'; // Default
+
+				break;
+
+			case 'uk' :
+
+				$subdomain = 'ua'; // Ukrainian (Українська)
+
+				break;
+
+			case 'el' :
+
+				$subdomain = 'gr'; // Greek (Ελληνικά)
+
+				break;
+
+		}
+
+		?>
+		<iframe src="<?php echo esc_url( "https://{$subdomain}.godaddy.com/help/managed-wordpress-1000021" ) ?>" frameborder="0" scrolling="no"></iframe>
```

Comment:
@EvanHerman This is not the right URL I think. We might want to send them to this one: https://www.godaddy.com/help/godaddy-email-marketing-1000013#nav-1. Don't forget to update the unit test 😉